### ISSUE TITLE
Convert NSwagStudio to MAUI for Mac Support

### ISSUE THREAD

**Issue by adrianvince7** [RicoSuter/NSwag#5006]

**Comment by marinasundstrom** [comment]
I don't think this is of any priority because of the state of the NSwag project. 
You are free to do so if you want to. But I would suggest doing it with Avalonia instead, since that probably is a more easy transition.

**Comment by adrianvince7** [comment]
I am experienced in MAUI and XamarinForms. I'll attempt a conversion to Avalonia and revert.

**Comment by RicoSuter** [comment]
Was also thinking about this but probably it would be a complete rewrite of the UI and then we would also improve the config file itself etc. and then yes, we should only do based on the long-term goal of NSwag which is probably going to be AspNetCore.OpenApi extensions + code generation.

**Comment by adrianvince7** [comment]
I think that the tool remains relevant until there is a mini of the config file. 

Hope to put in some more hours this week
